After Upgrade to v7 Agent and Enabling statements Plugin - "Error while fetching statements from instance: Internal error."

after upgrading and activating the `statements` plugin, we are receiving the following:

![image](https://user-images.githubusercontent.com/30243584/95785327-29761200-0c9b-11eb-8a67-a68ed81e53c2.png)

here's the relevant error in the temboard agent log file; i replaced the actual database name with "thedatabase":

```
2020-10-12 14:53:58,538 temboardagent[4757]: [probes] INFO: Loaded probe: heap_bloat
2020-10-12 14:53:58,538 temboardagent[4757]: [probes] INFO: Loaded probe: loadavg
2020-10-12 14:53:58,538 temboardagent[4757]: [probes] INFO: Loaded probe: filesystems_size
2020-10-12 14:53:58,539 temboardagent[4757]: [probes] INFO: Loaded probe: cpu
2020-10-12 14:53:58,539 temboardagent[4757]: [probes] INFO: Loaded probe: bgwriter
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR: thedatabase
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR: Traceback (most recent call last):
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:   File "/usr/lib/python2.7/site-packages/temboardagent/httpd.py", line 132, in response
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:     (code, message) = self.route_request()
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:   File "/usr/lib/python2.7/site-packages/temboardagent/httpd.py", line 286, in route_request
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:     return (200, func(http_context, self.app))
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:   File "/usr/lib/python2.7/site-packages/temboardagent/plugins/statements/__init__.py", line 31, in get_statements
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR:     assert dbname == "postgres", dbname
2020-10-12 14:54:19,235 temboardagent[12368]: [httpd] ERROR: AssertionError: thedatabase
2020-10-12 14:54:19,236 temboardagent[12368]: [httpd] ERROR: Internal error
2020-10-12 14:54:19,237 temboardagent[12368]: [httpd] INFO: client: pgstmb001v.prd.chi.accertify.net request: "GET /statements?key=9ae98817c598c1f6f34b110e80567b60 HTTP/1.1" 500 - 2.26ms
2020-10-12 14:54:23,444 temboardagent[12368]: [api] INFO: Starting discovery.
2020-10-12 14:54:23,466 temboardagent[12368]: [api] INFO: Discovery done.
2020-10-12 14:54:23,467 temboardagent[12368]: [httpd] INFO: client: pgstmb001v.prd.chi.accertify.net request: "GET /discover HTTP/1.1" 200 - 22.89ms
2020-10-12 14:54:23,523 temboardagent[12368]: [httpd] INFO: client: pgstmb001v.prd.chi.accertify.net request: "GET /monitoring/history?key=9ae98817c598c1f6f34b110e80567b60&limit=100&start=2
```

and here's our statements section in the `temboard-agent.conf` file:

```
[statements]
# Statements plugin part.
# DB name hosting pg_stat_statements view (the one where the extension has
# been created with "CREATE EXTENSION")
dbname = thedatabase
```

--- a/temboardagent/plugins/statements/__init__.py
+++ b/temboardagent/plugins/statements/__init__.py
@@ -28,7 +28,6 @@ def get_statements(http_context, app):
     """
     config = app.config
     dbname = config.statements.dbname
-    assert dbname == "postgres", dbname
     snapshot_datetime = now()
     conninfo = dict(config.postgresql, dbname=dbname)
     try:

